cmd/cluster/health.go

```diff
@@ -164,7 +176,7 @@ func (o *healthOptions) run() error {
 //This command implements the ocm describe clsuter call via osm-sdk.
 //This call requires the ocm API Token https://cloud.redhat.com/openshift/token be available in the OCM_TOKEN env variable.
 //Example: export OCM_TOKEN=$(jq -r .refresh_token ~/.ocm.json)
-func ocmDescribe(clusterID string) ([]string, string, int, error) {
+func ocmDescribe(clusterID string) ([]string, string, int, int, int, int, int, error) {
```

Comment:
I think with this many return parameters we should refactor this to be a specific struct, in order to make this more readable.

Maybe something like:

```
struct clusterDescribeOutput {
  az []string
  clusterName string
  compute int
  infra int
  master int
  autoscaleMin int
  autoscaleMax int
}
```
etc.

Reply:
If you want some inspiration, see https://github.com/openshift/osdctl/pull/129/commits/4333c8aa3d2ab63634512fe93808dd7e530c4390

Reply:
Thanks!!